main.cpp

```diff
@@ -654,7 +864,43 @@ int main()
     delta306.takeoff();
     delta306.turn(90.1f);
     float flushSpinDirection = delta306.pingRadar();
-    std::cout << "Here's your " << std::to_string(flushSpinDirection) << std::endl;
+    std::cout << "Spin this way: " << std::to_string(flushSpinDirection) << std::endl;
+
+    myPhone.screenShrinker(15);
+
+    float temp = jerrysKitchen.addBodyHeat();
+    std::cout << "New temp: " << std::to_string(temp) << std::endl;
+
+    jerrys.loseRating(5);
+
+    bool neighborIsAngry = thisOldAmp.checkIfNeighborsAngry(70.0f);
+    std::string neighborIsAngryString = neighborIsAngry ? "true" : "false";
+    std::cout << "Are you angry? " << neighborIsAngryString << std::endl;
+
+    float moneyStolen = chase.robbery("buff guy");
+    std::cout << "$" << std::to_string(moneyStolen) << " lost" << std::endl;
+
+    leftWing.breakRib();
+
+    pilotsPad.removeFamily(5, "reasonable");
+
+    float coffeeLevel = jessica.pourCoffee(5.0f, 25);
+    std::cout << "coffeeleft: " << std::to_string(coffeeLevel) << std::endl;
+    
+    float angle = passengersClub.reclineSeat(40.0f, true);
+    std::cout << "This far back: " << std::to_string(angle) << std::endl;
+
+    bool ladiesRoomStatus = ladiesRoom.callMaintenence(15.0f, 10);
+    std::string ladiesRoomStatusString = ladiesRoomStatus ? "true" : "false";
+    std::cout << "Bathroom status: " << ladiesRoomStatusString << std::endl;
+
+    bool roomLeft = jetOne.addFluid(0.0f, 20, 10.0f);
+    std::string roomLeftString = roomLeft ? "true" : "false";
+    std::cout << "Room for fuel: " << roomLeftString << std::endl;
+
+    bool stillFlying = delta306.checkFlightStatus(13);
+    std::string stillFlyingString = stillFlying ? "true" : "false";
```

Comment:
Prefer the use of `auto` instead of `float,bool,int, etc` when capturing values returned from functions, instead of the explicit type unless you specifically need to define the returned type (which may cause the compiler to perform an implicit cast)
- [ ] revise